kube-core/src/discovery.rs

```diff
@@ -107,3 +107,100 @@ impl ApiCapabilities {
         self.operations.iter().any(|op| op == operation)
     }
 }
+
+// Simple pluralizer. Handles the special cases.
+pub(crate) fn to_plural(word: &str) -> String {
```

Comment:
I don't think it's used anywhere else in `kube-core`.
```suggestion
fn to_plural(word: &str) -> String {
```

Reply:
Ah, good catch.